Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/README.md

```diff
@@ -1,3305 +0,0 @@
-Prisma™ Cloud Compute Edition delivers cloud workload protection (CWPP) for modern
```

Comment:
revert the deletion of this file and add the new command to the readme. this can be done using our [demisto-sdk generate-docs](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md) command.